components/app/explore/explore-dataset-filters/explore-dataset-filters.js

```diff
@@ -0,0 +1,33 @@
+import { Component, createElement } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import * as actions from './explore-dataset-filters-actions';
+import reducers, { initialState } from './explore-dataset-filters-reducer';
+
+import ExploreDatasetFiltersComponent from './explore-dataset-filters-component';
+import getFilterStatus from './explore-dataset-filters-selectors';
+
+const mapStateToProps = state => ({
+  data: getFilterStatus(state)
+});
+
+class ExploreDatasetFiltersContainer extends Component {
+  componentWillMount() {
```

Comment:
You are requesting filters here, I think you have to use componenDidMount. Are you using server rendering to get filters? In that case, you don't need it.